modules/lotamePanoramaIdSystem.js

```diff
@@ -18,6 +18,8 @@ const DAYS_TO_CACHE = 7;
 const DAY_MS = 60 * 60 * 24 * 1000;
 const MISSING_CORE_CONSENT = 111;
 const GVLID = 95;
+const ID_HOST = 'id.crwdcntrl.net';
+const DEV_ID_HOST = 'id.dev.lotame.com';
```

Comment:
Only Dev? not qa?

Reply:
That's a good point. I was about to decline this PR in favor of one with more fixes, plus latest from Prebid master, but I'll fold that in.